packages/modular-scripts/src/cli.ts

```diff
@@ -118,6 +128,38 @@ function run() {
   }
 }
 
+interface WorkspaceMeta {
+  location: string;
+  workspaceDependencies: string[];
+  mismatchedWorkspaceDependencies: string[];
+}
+
+function listPackages(typeArg: string | void) {
+  const modularRoot = getModularRoot();
+
+  const { stdout } = execa.sync('yarnpkg', ['workspaces', 'info'], {
+    cwd: modularRoot,
+  });
+  const workspaceInfo: Record<string, WorkspaceMeta> = JSON.parse(
+    stdout,
+  ) as Record<string, WorkspaceMeta>;
```

Comment:
Instead of a typecast here, I think it'd be better to use a type assertion (https://github.com/microsoft/TypeScript/pull/32695) to ensure it throws errors (perhaps when using a newer Yarn version?)